cmd/edit/firmware_set.go

```diff
@@ -5,11 +5,12 @@ import (
 	"log"
 
 	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+	ss "go.hollow.sh/serverservice/pkg/api/v1"
```

Comment:
Can you create a Jira ticket to move mctl's dependency here to FleetDB if such a ticket doesn't already exist?

Reply:
Created FS-1261